Source/WebCore/Modules/reporting/ReportingObserver.cpp

```diff
@@ -41,15 +41,17 @@ namespace WebCore {
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(ReportingObserver);
 
-static bool isVisibleToReportingObservers(const AtomString& type)
+static bool isVisibleToReportingObservers(const String& type)
 {
     static NeverDestroyed<Vector<AtomString>> visibleTypes(std::initializer_list<AtomString> {
```

Comment:
This function is not necessarily called from the main thread, correct?

If so, I don't think this static vector containing AtomStrings is actually safe.

Reply:
Ah, okay.